### PR TITLE
WIP: Added a more descriptive output for the runs.

### PR DESCRIPTION
**Pull request by pedro93**
Addresses https://github.com/paulirish/pwmetrics/issues/64. Very minor code change.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -53,7 +53,11 @@ class PWMetrics {
     const allDone = results.reduce((chain, run, index) => {
       return chain.then(_ => {
         return this.run().then(data => {
+          console.log("Run",index+1 , "of", results.length, "finished successfully");
           results[index] = data;
+        }).catch(error => {
+          console.log("unable to complete run", index+1, "of", results.length, "due to", error.message);
```
Comment (paulirish) [inline lib/index.js]: can you change to use template strings?

Comment (denar90) [inline lib/index.js]: + can you move text into messages and use `getMessage` for this?

**Comment by pedro93** [comment]
Here is the updated output, @rpkoller is this better for you?

screen shot 2017-03-08 at 12 36 32

File: lib/index.js (hunk 1 above)
Comment (pedro93) [inline lib/index.js]: Done both, please see new commit and program output in the screenshot I commented :)

**Comment by rpkoller** [comment]
Looks great! The only thing i am still unsure if `Launching Chrome`is necessary at all since you see the opening chrome window anyway.

File: lib/utils/messages.js [hunk 2]
```diff
@@ -13,56 +13,60 @@ const redify = str => `${RED}${str}${RESET}`;
...
+    case 'vc':
+      return 'Visually Complete 100%';
+    case 'tti':
+      return 'Time to Interactive';
+    case 'vc85':
+      return 'Visually Complete 85%';
+    case 'SUCCESS_RUN':
+      return `Run ${args[0][0] + 1} of ${args[0][1]} finished successfully`;
+    case 'FAILED_RUN':
+      return `Unable to complete run ${args[0][0] + 1} of ${args[0][1]} due to ${args[0][2]}`
```
Comment (pedro93) [inline lib/utils/messages.js]: Never worked with rest parameters before, this was the way I found to access the variable data needed. Is it correct?

Comment (denar90) [inline lib/utils/messages.js]: Just do smth like
`Run ${args[0] + 1} of ${args[1]} finished successfully`

**Comment by denar90** [comment]
We are launching/killing chrome for each run. It's useful to have this message when errors happens or for CI as @pedro93 noticed.

File: lib/utils/messages.js (hunk 2 above)
Comment (pedro93) [inline lib/utils/messages.js]: Args[0] is an array like this: [index, results.length] e.g: [1,3] with args being [[1,3]], not sure why though. I have a suspicion it's the nested rest parameters generated by getmessagewithprefix

Comment (pedro93) [inline lib/utils/messages.js]: Here is quick (it's bad don't judge) code snippet demonstrating what was happening: 
https://repl.it/GNlK/0

 I fixed it in this [commit](https://github.com/paulirish/pwmetrics/pull/70/commits/f2bd2e1afa5f697fced9d5e5c3c934bc6de1b809). Doesn't seem to break anything, can anyone confirm @paulirish @denar90 ?

**Comment by denar90** [comment]
@paulirish can we merge this one?

**Comment by pedro93** [comment]
@denar90 @paulirish with your permission I would merge both this PR and the eslint one tomorrow.

**Pull request merged**